crm_opportunity_currency/views/crm_lead_opportunity_currency_views.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+
+<odoo>
+
+  <record id="crm_case_form_view_oppor" model="ir.ui.view">
```

Comment:
Somehow these views should be restricted to group `base.group_multi_currency`

Reply:
I am not sure